internal/platform/cpuid_arm64.s

```diff
@@ -0,0 +1,21 @@
+//go:build gc
+
+#include "textflag.h"
+
+// lifted from github.com/golang/sys and cpu/cpu_arm64.s
+
+// func getisar0() uint64
+TEXT ·getisar0(SB), NOSPLIT, $0-8
+	// get Instruction Set Attributes 0 into x0
+	// mrs x0, ID_AA64ISAR0_EL1 = d5380600
```

Comment:
could you put the reference to the arm64 manual?

Reply:
If you mean the constants (`0xd5380600`), I have none besides Go code for `runtime`, `x/sys/cpu` and `klauspost/cpuid`. Other assemblers apparently get this from a (assembly) macro, that the Go assembler doesn't support.